app/domain/healthchecks/sidekiq_retry_size.rb

```diff
@@ -0,0 +1,11 @@
+module Healthchecks
+  class SidekiqRetrySize < GovukHealthcheck::SidekiqRetrySizeCheck
+    def critical_threshold
+      100
+    end
+
+    def warning_threshold
+      1
```

Comment:
Could this potentially be too low and become noise?

Reply:
Yep, I think we'll just have to see how it goes. Having an empty retry set is ideal, and it does seem to be the case at the moment.